atoma-proxy/src/server/handlers/nodes.rs

```diff
@@ -336,7 +337,7 @@ pub async fn nodes_create_lock(
             Ok(Json(NodesCreateLockResponse {
                 public_key,
                 node_small_id: node_public_key.node_small_id as u64,
-                stack_entry_digest: None,
+                stack_entry_digest: TransactionDigest::ZERO.base58_encode(),
```

Comment:
No, this is not good. It must be the actual exact transaction digest of the stack.

Reply:
Good point 24c7dcf